thingsboard_gateway/tb_utility/tb_utility.py

```diff
@@ -26,6 +27,7 @@
 from cryptography.hazmat.primitives import hashes, serialization
 from cryptography.hazmat.primitives.asymmetric import ec
 from cryptography.x509.oid import NameOID
+from distutils.util import strtobool
```

Comment:
It is not necessary to use this library, also it provokes issue with python 3.12+, because it is no longer built-in, please remove this change and use removed by you str_to_bool method instead.